### PR TITLE
Adding blog link in the leaderboard page

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
image

### CODE REVIEW

File: app/views/rating_leaderboard/index.html.erb [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <div class="row">
       <div class="col-12">
         <h1>AIcrowd Rankings</h1>
+        <%= link_to "Learn more about rankings", blog_path(10) %>
```
Comment (skbly7) [inline app/views/rating_leaderboard/index.html.erb:6]: Refactor it to link blog via slug, so we have control in future. Linking via ID isn't a good way to do so.

Comment (skbly7) [reply app/views/rating_leaderboard/index.html.erb:6]: Ideally, have that slug with some info about rating added in seed as well. (extended goal)

Comment (piotrekpasciak) [reply app/views/rating_leaderboard/index.html.erb:6]: Like @skbly7 mentioned. Using record `id` is very bad practice.

**Comment by nikhilrayaprolu** [comment]
@skbly7 @piotrekpasciak made requested changes

**Pull request merged**